app/views/oaisys/pmh/list_sets.xml.builder

```diff
@@ -23,7 +23,7 @@ xml.tag!('request', parameters, 'https://era.library.ualberta.ca/oai')
 xml.ListSets do
   sets.each do |full_set_id, full_set_name, description|
     xml.tag!('set') do
-      xml.setSpec full_set_id
+      set_id(xml: xml, id: full_set_id)
```

Comment:
really minor, but if this is the only place we need it we can probably avoid the helper and just do it inline, ie)

```ruby
xml.setSpec(id) unless id.blank?
```

If we need this in multiple places then it would totally make sense, but it's a layer of indirection that's probably unnecessary given that the underlying implementation is so simple that it's basically just as clean to do it in-place in the only spot it's needed.